geniza/corpus/templates/admin/corpus/document/snippets/mixed_inlines_fieldsets.html

```diff
@@ -0,0 +1,20 @@
+{% load admin_extras %}
+
+{% comment %}
+workaround to reposition inlines by Bertrand Bordage:
+https://github.com/dezede/dezede/commit/ed13cc
+{% endcomment %}
+
+{# Render mixed normal and inline fieldsets #}
+{% get_fieldsets_and_inlines as fieldsets_and_inlines %}
+{% for type, fieldset_or_inline in fieldsets_and_inlines %}
```

Comment:
interesting. seems kind of complicated and a bit hacky, but if it's tested and working just make sure there are brief comments everywhere it's used so we can find it again when we need to remember what it's doing or make changes